kolena/annotation.py

```diff
@@ -190,6 +191,9 @@ class Keypoints(Annotation):
     points: List[Tuple[float, float]]
     """The sequence of discrete `(x, y)` pixel coordinates comprising this keypoints annotation."""
 
+    skeleton: Optional[List[Tuple[float, float]]] = None
+    """A sequence of paired indices indicating connections between `points`."""
```

Comment:
The reason these were left off initially is because dataclasses with optional fields don't play nicely with subclassing. This change will break any usages like this:

```py
from dataclasses import dataclass
from kolena.annotation import Keypoints

@dataclass(frozen=True)
class MyKeypoints(Keypoints):
    my_field: str
```

The above script runs on the latest release, but on this branch fails with:

```
TypeError: non-default argument 'my_field' follows default argument
```